backend/routes/group.py

```
@@ -48,3 +53,81 @@
 
     except Exception as e:
         raise HTTPException(status_code=500, detail=str(e))
+
+
+@router.get("/", response_model=List[UniversityListResponse])
```

Ruff (0.8.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)